x-pack/plugins/canvas/canvas_plugin_src/functions/common/joinRows.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ExpressionFunction } from '../../../../../../src/legacy/core_plugins/interpreter/types';
+import { Datatable } from '../types';
+import { getFunctionErrors } from '../../strings/functions';
```

Comment:
We want to import the help text for the function from `/strings/functions/` for i18n support.
```suggestion
import { getFunctionHelp, getFunctionErrors } from '../../strings';
```